### DISCUSSION TITLE
Dashcam Viewer Integration

### DISCUSSION THREAD

**Discussion by MehdiBrahmi** [teslamate-org/teslamate#4434] [Ideas]
Hi team,

Now that the Dashcam Viewer option is available in the Tesla app on iPhone, do you think it would be technically possible to leverage the related API endpoints to create an interface within TeslaMate for viewing these dashcam clips?

Looking forward to your insights!

**Comment by brianmay** [comment]
My gut reaction is this is perhaps outside the scope of Teslamate, which is mostly designed to run in the background in a non-interactive manner.

Sounds like a really good idea for a new project however.
